### PR TITLE
Reporting radio buttons

### PR DESCRIPTION
**Pull request by phoebemirman**
part of https://github.com/AugurProject/augur/issues/2912

### CODE REVIEW

File: packages/augur-ui/src/modules/common/form.tsx [hunk 1]
```diff
@@ -691,7 +692,16 @@ export const ReportingRadioBar = ({
     {checked ? FilledRadio : EmptyRadio}
     <h5>{header}</h5>
     <div>
-      <ReportingPercent firstPercent={20} firstPercent={40} firstPercent={30} />
+      {!stake.tentativeWinning &&
+        <>
+          <ReportingPercent firstPercent={stake.preFilledStake} secondPercent={stake.bondSizeCurrent} thirdPercent={formatRep(0)} total={stake.bondSizeTotal} />
```
Comment (bthaile) [inline packages/augur-ui/src/modules/common/form.tsx:710]: is there a reason the `thirdPercent` is set to formatRep(0)

File: packages/augur-ui/src/modules/market-cards/common.tsx [hunk 2]
```diff
@@ -84,7 +84,7 @@ export interface ScalarOutcomeProps {
   lastPrice?: FormattedNumber;
 }
 
-export function calculatePosition(min, max, lastPrice) {
+export function calculatePosition(min: BigNumber, max: BigNumber, lastPrice: FormattedNumber) {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market-cards/common.tsx:87]: love the use of `BigNumber`

File: packages/augur-ui/src/modules/common/form.tsx (hunk 1 above)
Comment (phoebemirman) [reply packages/augur-ui/src/modules/common/form.tsx:710]: that is temporary. its the one that is going to come from the text input

File: packages/augur-ui/src/modules/reporting/common.tsx [hunk 3]
```diff
@@ -1,19 +1,60 @@
...
+          id='tooltip--existingStake'
+          className={TooltipStyles.Tooltip}
+          effect='solid'
+          place='top'
+          type='light'
+        >
+         	My Existing Stake
+          	<p>{props.firstPercent.formattedValue} REP</p>
+        </ReactTooltip>
+	  	<span style={{width: thirdPercent + "%", left: firstPercent + thirdPercent + 'px'}}/>
```
Comment (bthaile) [inline packages/augur-ui/src/modules/reporting/common.tsx]: can we use ${} syntax instead of `+` here? Yes, super nit pic

**Review by bthaile — APPROVED** [review]
lgtm, small comment

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🎇

 This link will always point to this PR's latest changes: https://peek.run/p37hp96

File: packages/augur-ui/src/modules/reporting/common.tsx [hunk 4]
```diff
@@ -1,21 +1,62 @@
...
+  total: BigNumber;
 }
 
-export const ReportingPercent = (props: ReportingPercentProps) => (
-  <div className={Styles.ReportingPercent}>
-  	<span style={{width: props.firstPercent + "%"}}></span>
-  	<span style={{width: props.secondPercent + "%"}}></span>
-  	<span style={{width: props.thirdPercent + "%"}}></span>
+export const ReportingPercent = (props: ReportingPercentProps) => {
+	const firstPercent = calculatePosition(ZERO, createBigNumber(props.total.value), props.firstPercent);
```
Comment (bthaile) [inline packages/augur-ui/src/modules/reporting/common.tsx:21]: this looks weird to me. If `total` is a BigNumber why is it being used as `props.total.value` like a FormattedNumber?

Comment (phoebemirman) [reply packages/augur-ui/src/modules/reporting/common.tsx:21]: it is a FormattedNumber

**Review by bthaile — APPROVED** [review]
lgtm

**Pull request merged**